modules/monitoring-configuring-persistent-storage.adoc

```diff
@@ -13,7 +13,7 @@ Running cluster monitoring with persistent storage means that your metrics are s
 
 * Dedicate sufficient local persistent storage to ensure that the disk does not become full. How much storage you need depends on the number of pods.
 
-* Make sure you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has three replicas, you need five PVs to support the entire monitoring stack. The PVs should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
+* Make sure you have a persistent volume (PV) ready to be claimed by the persistent volume claim (PVC), one PV for each replica. Because Prometheus has two replicas and Alertmanager has two replicas, you need four PVs to support the entire monitoring stack. The PVs should be available from the Local Storage Operator. This does not apply if you enable dynamically provisioned storage.
```

Comment:
Not a necessary change, but when you say `PVS _should_ be available` it implies they _might not_ be available. So you can do three things:

1. leave as is, because this content is clear and concise. 
2. rewrite the sentence if you are confident the PVS are there: `The PVs are available from the Local STorage Operator.` 
3. Explain the circumstance if the PVs _are not_ there: `The PVs should be available from the Local Storage Operator. If the PVs are not in available from the Local Storage Operator, go to <where users must go>` OR ` If the PVs are not in available from the Local Storage Operator, then you cannot claim a PV for each replica <or whatever the consequence is>`